pepper-apis/docs/specification/src/components/responses.yml

```diff
@@ -76,7 +76,7 @@ ActivityAnswersPatchBadPayloadResponse:
       schema:
         $ref: '../pepper.yml#/components/schemas/Error.ActivityAnswersPatchBadPayload'
 ActivityAnswersPatchInvalidResponse:
-  description: activity instance is read-only or answer submission failed validation
+  description: activity instance or question is read-only or answer submission failed validation
```

Comment:
The question's readonly property also needs to be documented in the schemas for the Question objects.